api/v1alpha1/selfnoderemediation_types.go

```diff
@@ -23,8 +23,10 @@ import (
 const (
 	ResourceDeletionRemediationStrategy  = RemediationStrategyType("ResourceDeletion")
 	OutOfServiceTaintRemediationStrategy = RemediationStrategyType("OutOfServiceTaint")
-	// SnrConditionProcessing is the condition type used to signal NHC the remediation status
-	SnrConditionProcessing = "Processing"
+	// ProcessingTypeCondition is the condition type used to signal NHC the remediation status
+	ProcessingTypeCondition = "Processing"
```

Comment:
somehow this reads a bit strange... would either ProcessingConditionType or ConditionTypeProcessing be a better fit?